lib/CoreBot.js

```diff
@@ -1488,6 +1488,15 @@ function Botkit(configuration) {
         botkit.config.port = port;
 
         botkit.webserver = express();
+        botkit.webserver.use(function(req, res, next) {
```

Comment:
Can you make this same change to the slack starter kit?  github.com/howdyai/botkit-starter-slack

Reply:
Sure